### PR TITLE
GraphQL: Migrate Quote endpoint topics

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) moves all cart-related mutation and query topics from the `graphql/reference` directory to the `mutations` and `queries` directories. The topics were also renamed to more closely resemble mutation or query name.

## Affected DevDocs pages

- Numerous

### CODE REVIEW

File: _includes/graphql/payment-methods/payflow-link-workflow.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
-1. The PWA client uses the [`setPaymentMethodOnCart`]({{page.baseurl}}/graphql/reference/quote-payment-method.html) mutation to set the payment method.
+1. The PWA client uses the [`setPaymentMethodOnCart`]({{page.baseurl}}/graphql/mutations/set-payment-method.html) mutation to set the payment method.
 
 1. The mutation returns a `Cart` object.
 
-1. The client runs the [`placeOrder`]({{page.baseurl}}/graphql/reference/quote-place-order.html) mutation, which creates an order in Magento and begins the authorization process.
+2. The client runs the [`placeOrder`]({{page.baseurl}}/graphql/mutations/place-order.html) mutation, which creates an order in Magento and begins the authorization process.
```
Comment (erikmarr) [inline _includes/graphql/payment-methods/payflow-link-workflow.md]: Did these numbers get sequentialized again?

File: _includes/graphql/payment-methods/hosted-pro-workflow.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
-1. The PWA client uses the [`setPaymentMethodOnCart`]({{page.baseurl}}/graphql/reference/quote-payment-method.html) mutation to set the payment method.
+1. The PWA client uses the [`setPaymentMethodOnCart`]({{page.baseurl}}/graphql/mutations/set-payment-method.html) mutation to set the payment method.
 
 1. The mutation returns a `Cart` object.
 
-1. The client runs the [`placeOrder`]({{page.baseurl}}/graphql/reference/quote-place-order.html) mutation, which creates an order in Magento and begins the authorization process.
+2. The client runs the [`placeOrder`]({{page.baseurl}}/graphql/mutations/place-order.html) mutation, which creates an order in Magento and begins the authorization process.
```
Comment (erikmarr) [inline _includes/graphql/payment-methods/hosted-pro-workflow.md]: Did these numbers get sequentialized again?

File: guides/v2.3/graphql/mutations/create-empty-cart.md [hunk 3]
```diff
@@ -68,4 +60,14 @@ The mutation returns the same `cart_id`.
...
+
+## Input attributes
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`cart_id` | String | An optional 32-character string
+
+## Output attributes
+
+The `createEmptyCart` mutation returns a string, the cart ID.
```
Comment (erikmarr) [inline guides/v2.3/graphql/mutations/create-empty-cart.md]:
```suggestion
The `createEmptyCart` mutation returns a string and the cart ID.
```

File: guides/v2.3/graphql/release-notes.md [hunk 4]
```diff
@@ -17,60 +17,60 @@ These release notes can include:
...
-- {:.new} **Added new queries and extended the functionality of others.**
+-  {:.new} **Added support for payment methods that implement [Magento Vault]({{page.baseurl}}/graphql/reference/vault.html)**
 
-  - The [`isEmailAvailable` query]({{page.baseurl}}/graphql/reference/customer.html) checks whether the specified email address has already been used to create an account.
-  - The [`cart` query]({{page.baseurl}}/graphql/reference/quote.html) can now return information set by mutations that perform cart operations, including product information, shipping and billing addresses, shipping and payment methods, and applied coupons. The query also returns calculated totals.
-  - The `customerPaymentTokens` query returns the signed-in customer's payment tokens.
+-  {:.new} **Added new queries and extended the functionality of others.**
 
-- {:.new} **Queries can now be performed as HTTP GET or POST operations.**
+   -  The [`isEmailAvailable` query]({{page.baseurl}}/graphql/reference/customer.html) checks whether the specified email address has already been used to create an account.
```
Comment (erikmarr) [inline guides/v2.3/graphql/release-notes.md]: Should this link point to https://devdocs.magedevteam.com/804/guides/v2.3/graphql/queries/customer.html?

Comment (keharper) [reply guides/v2.3/graphql/release-notes.md]: Fixed to point to `queries/is-email-available.html`

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
running tests

**Pull request merged**